src/utils.ts

```diff
@@ -9,18 +9,24 @@ import * as semver from 'semver';
 export class Utils {
     // eslint-disable-next-line @typescript-eslint/no-var-requires
     public static readonly USER_AGENT: string = 'setup-jfrog-cli-github-action/' + require('../package.json').version;
-    public static readonly SERVER_TOKEN_LEGACY_PREFIX: RegExp = /^JF_ARTIFACTORY_.*$/;
-    public static readonly SERVER_TOKEN_PREFIX: RegExp = /^JF_ENV_.*$/;
-    // Since 1.45.0, 'jfrog rt c' command changed to 'jfrog c add'
-    public static readonly NEW_CONFIG_CLI_VERSION: string = '1.45.0';
-    public static readonly CLI_VERSION_ARG: string = 'version';
-    public static readonly MIN_CLI_VERSION: string = '1.29.0';
-    public static readonly LATEST_CLI_VERSION_ARG: string = 'latest';
+    public static readonly DEFAULT_DOWNLOAD_DETAILS: DownloadDetails = {
+        artifactoryUrl: 'https://releases.jfrog.io/artifactory',
+        repository: 'jfrog-cli',
+    } as DownloadDetails;
 
+    private static readonly SERVER_TOKEN_LEGACY_PREFIX: RegExp = /^JF_ARTIFACTORY_.*$/;
+    private static readonly SERVER_TOKEN_PREFIX: RegExp = /^JF_ENV_.*$/;
+    // Since 1.45.0, 'jfrog rt c' command changed to 'jfrog c add'
+    private static readonly NEW_CONFIG_CLI_VERSION: string = '1.45.0';
+    private static readonly CLI_VERSION_ARG: string = 'version';
+    private static readonly MIN_CLI_VERSION: string = '1.29.0';
+    private static readonly LATEST_CLI_VERSION_ARG: string = 'latest';
     private static readonly LATEST_RELEASE_VERSION: string = '[RELEASE]';
+    private static readonly CLI_REMOTE_ARG: string = 'download-repository';
```

Comment:
I suggest we add a description for all of the above config properties. This should make the code easier to read for the random reader.